src/utils/logger.js

```diff
@@ -0,0 +1,32 @@
+require('dotenv').config();
```

Comment:
what does this line do?

Reply:
This line is to require and configure dotenv. so I can get loglvl from .env file